src/components/Link/Link.scss

```diff
@@ -18,6 +19,26 @@
   }
 }
 
+// Prevents the icon from wrapping on its own to the next line
```

Comment:
I think these comments can go, it's pretty self explanatory. Could even call this `ExternalIconLockUp` if we want more clarity.